packages/dpcpp/package.py

```diff
@@ -41,20 +48,28 @@ class Dpcpp(Package):
     SYCL. This involves setting some extra environment variables for
     the Intel oneAPI compilers."""
 
-    homepage = "https://software.intel.com/content/www/us/en/develop/tools/oneapi.html"
+    homepage = (
+        "https://software.intel.com/content/www/us/en/develop/tools/oneapi.html"
+    )
     maintainers = ["cmacmackin"]
 
+    # required to populate self.compiler.cxx?
+    depends_on("c")
+    depends_on("cxx")
+    
+    version("working")
```

Comment:
Why do we need this here? The idea when I originally introduced the concept of a "working" version was that it would be the development branch. We aren't using a dev branch fro dpcpp are we?

(Seeing as we don't actually seem to have a dev branch for any of our project, I'd prefer to get rid of the "working" version now, although that's for another PR).